source/guide_adminer.rst

```diff
@@ -0,0 +1,140 @@
+.. highlight:: console
+
+.. author:: Nikolas <https://nikolasdas.de>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: database
+
+.. sidebar:: About
+
+  .. image:: _static/images/adminer.svg
+      :align: center
+
+#######
+Adminer
+#######
+
+.. tag_list::
+
+Adminer_ (formerly phpMinAdmin) is a full-featured database management tool written in PHP. Conversely to phpMyAdmin, it consist of a single file ready to deploy to the target server. Adminer is available for :manual:`MySQL <database-mysql>`, MariaDB, :lab:`PostgreSQL <guide_postgresql>`, SQLite, MS SQL, Oracle, Firebird, SimpleDB, Elasticsearch and :lab:`MongoDB <guide_mongodb>`.
+
+----
+
+.. note::
+
+  Uberspace provides instances of Adminer_ and phpMyAdmin_ for everbody to use. Therefore this guide is mainly for people who would like to customize Adminer to there needs.
+
+  * Adminer: https://mysql.uberspace.de/adminer/
+  * phpMyAdmin: https://mysql.uberspace.de/phpmyadmin/
+
+
+License
+=======
+
+Adminer_ is released under `Apache License 2.0`_ or `GPL 2`_.
+
+Installation
+============
+
+Check the current version of Adminer at `GitHub <https://github.com/vrana/adminer/releases>`_ and go the the folder where you want Adminer to be installed
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ VERSION=4.7.8
+  [isabell@stardust ~]$ cd ...
+  [isabell@stardust ~]$
+
+Option 1: Install Adminer without Plugin Support
+------------------------------------------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ wget -O index.php https://github.com/vrana/adminer/releases/download/v$VERSION/adminer-$VERSION.php
+  [isabell@stardust ~]$
+
+That's it!
+
+Option 2: Install Adminer with Plugin Support
+---------------------------------------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ wget -O adminer.php https://github.com/vrana/adminer/releases/download/v$VERSION/adminer-$VERSION.php
+  [isabell@stardust ~]$ mkdir -p plugins
+  [isabell@stardust ~]$ wget -O plugins/plugin.php https://raw.githubusercontent.com/vrana/adminer/master/plugins/plugin.php
+  [isabell@stardust ~]$
+
+Create an ``index.php`` file:
+
+.. code-block:: php
+
+  <?php
+
+  function adminer_object() {
+    foreach (glob("plugins/*.php") as $filename) {
+      include_once "./$filename";
+    }
+
+    $plugins = array(
+      // ...
+    );
+
+    return new AdminerPlugin($plugins);
+  }
+
+  include "./adminer.php";
+
+This will include all plugins located in the ``plugins`` folder. To use a plugin you also need to initialize it inside the ``$plugins`` array.
+
+You can find a list of available Plugins `here <https://www.adminer.org/plugins/>`_.
+
+Example Plugin: Themes
```

Comment:
What do you think of only providing this section, skipping the theoretical "Install Adminer with Plugin Support" one and the "Install Adminer without Plugin Support" one? Users, who don't need plugins, can use the adminer provided by us. Users can adapt the theme example to their needs. It would also keep the guide nice and short, without any duplicated code or commands.

Reply:
Valid point. I removed the part without plugins